src/_guides/language/sound-dart.md

```diff
@@ -507,7 +515,15 @@ For more information on these flags, see
 
 If you use [DartPad](/tools/dartpad) to write and test code, you can
 enable strong mode by selecting the **Strong mode** box in the lower
-right corner. Note that DartPad doesn't support the implicit casts flag,
+right corner.
+
+<aside class="alert alert-warning" markdown="1">
+  **Warning:**
+  While DartPad implements strong mode _static checks_,
+  DartPad does **not** currently implement strong mode _runtime checks_.
+</aside>
```

Comment:
@eernstg - how's this as a warning about the partial strong mode support in DartPad?  (FYI, I deliberately chose not to mention "Dart 1 runtime".)

Reply:
I like it!  But maybe changing `currently` to `yet` would induce more hope, because it implies that DartPad definitely will support those checks.

Reply:
Yes! Done.